go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/ozonru/sarama-keycloak
+
+go 1.14
+
+require (
+	github.com/Nerzal/gocloak v1.0.0
```

Comment:
Why don't you use `github.com/Nerzal/gocloak/v5@latest`? It seems backward compatible with some nice (maybe) additions.

Reply:
updated